newIDE/app/src/ObjectsList/index.js

```diff
@@ -110,7 +110,7 @@ class ObjectsList extends Component<*, *> {
               onCutObject={() => this.props.onCutObject(objectWithContext)}
               onPaste={() => this.props.onPaste(objectWithContext)}
               onRename={newName =>
-                this.props.onRename(objectWithContext, newName)}
+                this.props.onRename(objectWithContext, newName)}            
```

Comment:
nitpicking: can you remove these extra spaces?

Reply:
removed now